docs/recipes.md

```diff
@@ -0,0 +1,58 @@
+<!--
+SPDX-FileCopyrightText: 2023 Springtime authors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+We provide several "recipes" for downloading data from various sources and
+executing a range of ML methods with it.
+Note that a "recipe" is a file with `yaml` extension.
+
+## Datasets
+
+??? example "Daymet recipe"
```

Comment:
These foldable boxes are really nice! I just wonder if they should be presented as "example use cases" or rather as part of the "datasets" documentation.

Reply:
moved to `Datasets` page. However, the `Datasets` page re-organized in the future considering API usage and recipe usage.